src/ios/APPNotificationOptions.m

```diff
@@ -160,32 +160,21 @@ - (NSString*) actionGroupId
 
 /**
  * The sound file for the notification.
- *
+ * Sound option (string) can be given in options.
  * @return [ UNNotificationSound* ]
  */
 - (UNNotificationSound*) sound
```

Comment:
so passing the string `komed_notification_default.wav` didn't work here?

Reply:
that would work sure, but then we would need to pass it and change the code on FE and komed plugin. This way only one code base is needs to be changed (here). I could make it dynamic also and just use default if fileName is not provided. This was just the fastest way to get the result.

Reply:
but to allow passing filename won't be a biggy so I can do that

Reply:
Okay it's now more dynamic as the method description also says - so it's possible to override the default sound by providing a filename